Std.UriTemplate.csproj

```diff
@@ -52,7 +52,21 @@
   <ItemGroup>
     <None Include="..\Readme.md">
       <Pack>True</Pack>
-      <PackagePath></PackagePath>
+      <PackagePath>\</PackagePath>
+    </None>
+
+    <None Include="LICENSE">
+      <Pack>True</Pack>
+      <PackagePath>\</PackagePath>

```

Comment:
```suggestion
```

We can use `<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>` in the property group instead of this. 
This makes automated audits/reviews easier in specific scenarios. And the package slightly smaller.

Reply:
Oh that's actually already included. My bad